src/snowflake/cli/plugins/nativeapp/codegen/setup/native_app_setup_processor.py

```diff
@@ -35,19 +35,15 @@
     SandboxEnvBuilder,
     execute_script_in_sandbox,
 )
-from snowflake.cli.plugins.nativeapp.project_model import NativeAppProjectModel
 from snowflake.cli.plugins.stage.diff import to_stage_path
 
 DEFAULT_TIMEOUT = 30
 DRIVER_PATH = Path(__file__).parent / "setup_driver.py.source"
 
 
 class NativeAppSetupProcessor(ArtifactProcessor):
-    def __init__(
-        self,
-        na_project: NativeAppProjectModel,
-    ):
-        super().__init__(na_project=na_project)
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
why change the contract? This will accept anything instead of erroring out if the wrong parameters are passed.

Reply:
It will fail on the base class if the wrong parameters are passed. I don't mind keeping the existing signature, but it's a duplicate of the base class and doesn't add anything to it.

Reply:
Yup, I read that wrong, we can keep as is.